tests/components/analytics_insights/test_sensor.py

```diff
@@ -27,6 +28,18 @@ async def test_all_entities(
     entity_registry: er.EntityRegistry,
 ) -> None:
     """Test all entities."""
+    unique_ids = ["total_active_installations", "total_reports_integrations"]
+
+    for unique_id in unique_ids:
+        entity_registry.async_get_or_create(
+            domain=Platform.SENSOR,
+            platform=DOMAIN,
+            unique_id=unique_id,
+            config_entry=mock_config_entry,
+            suggested_object_id=unique_id,
+            disabled_by=None,
+        )
+
```

Comment:
Please use the fixture to enable entities by default instead

Reply:
I did not know there was a fixture for this.
Very nice, thanks!